### PR TITLE
github.com/nlopes/slack/issues/216

### PR DESCRIPTION
**Pull request by apokalyptik**

### CODE REVIEW

**Comment by jyap808** [comment]
I've observed the issue mentioned in #216 issue for a month. I can confirm this fixes the endless loop issue and the bot works fine now.

**Comment by jeberly** [comment]
I can confirm it fixed a problem we had seen in production on several slack teams.

**Comment by dvdplm** [comment]
lgtm

**Comment by lox** [comment]
Would be great to get this merged!

File: rtm.go [hunk 1]
```diff
@@ -64,12 +59,7 @@ func (api *Client) ConnectRTMContext(ctx context.Context) (info *Info, websocket
 	// Fixed by: https://github.com/golang/net/commit/5058c78c3627b31e484a81463acd51c7cecc06f3
 	// but slack returns the address with no port, so we have to fix it
```
Comment (dvdplm) [inline rtm.go]: @apokalyptik can you please update this comment and remove the `websocketizeURLPort()` function?

**Comment by apokalyptik** [comment]
@dvdplm PR Updated, per request.

**Comment by dvdplm** [comment]
@apokalyptik thanks, great work here! :)

**Pull request merged**